arraysForDeletion.js

```diff
@@ -0,0 +1,246 @@
+
+        for (let i = 0;i<emptyTabCopy.length;i++){               // creating arrays with all left and right empty tabs
+            for (let j = 0;j<leftEdgesInt.length;j++){
+                if (emptyTabCopy[i]===leftEdgesInt[j]){
+                    arrayLeft.push(leftEdgesInt[j])            // array with only left empty tabs
+                }
+                else if (emptyTabCopy[i]===rightEdgesInt[j]){
+                    arrayRight.push(rightEdgesInt[j])
+                }
+            }
+        }
+        console.log(emptyTabCopy)
+        for (let i = 0;i<leftEdgesInt.length;i++){               // deleting right and left empty tabs (different logic would be made for them)
+            for (let j = 0;j<emptyTabCopy.length;j++){
+                if (emptyTabCopy[j]===leftEdgesInt[i]){
+                    emptyTabCopy.splice(emptyTabCopy.indexOf(emptyTabCopy[j]),1)
+                }
+                else if (emptyTabCopy[j]===rightEdgesInt[i]){
+                    emptyTabCopy.splice(emptyTabCopy.indexOf(emptyTabCopy[j]),1)
+                }
+            }
+        }
+        console.log(emptyTabCopy)
+
+        array1.push(emptyTabCopy[0])
+        for (let k =0;k<5;k++){                      // rechecking massive 5 times to make sure every number is correct (if not rechecked it might left some doubles)
+
+            array1 = array1.filter((item, index) => {          // deleting doubles
+                return array1.indexOf(item) === index
+            })
+            for (let i = 0;i<emptyTabCopy.length;i++){
+                for (let j=0;j<array1.length;j++){
+                    if (emptyTabCopy[i]===array1[j]+1 || emptyTabCopy[i]===array1[j]-board1.y || emptyTabCopy[i]===array1[j]-1 || emptyTabCopy[i]===array1[j]+board1.y){
+                        array1.push(emptyTabCopy[i])
+                    }
+
+
+                }
+            }
+        }
+
+        arrayLeft = arrayLeft.filter((item, index) => {
+            return arrayLeft.indexOf(item) === index
+        })
+        function tabsSeparation(tempArraySideSeparate,edges,array) {
+            for (let i = 0;i<array.length;i++){             // separating right and left empty fields and adding them into a different array
+                let limit = board1.x
+                if (array[i]===edges[edges.length-1]){
+                    limit = 0
+                }
+                let getDocumentNext = document.getElementById(board1.tableId[array[i]+limit])
+                tempArrayForSeparation.push(array[i])
+                if (getDocumentNext.innerHTML !== ' '){
+                    tempArraySideSeparate.push(tempArrayForSeparation)
+                    tempArrayForSeparation = []
+                }
+                else if (array[i]===edges[edges.length-1]){
+                    tempArraySideSeparate.push(tempArrayForSeparation)
+                    tempArrayForSeparation = []
+                }
+
+
+
+
+
```

Comment:
remove empty lines. Your code has to look good as well and be readable. You are not writing code to yourself.